app/src/main/java/org/mozilla/fenix/home/pocket/PocketCategoriesViewHolder.kt

```diff
@@ -103,10 +104,7 @@ class PocketCategoriesViewHolder(
 
 @Composable
 private fun PocketTopics(
-    selectedTextColor: Color? = null,
-    unselectedTextColor: Color? = null,
-    selectedBackgroundColor: Color? = null,
-    unselectedBackgroundColor: Color? = null,
+    categoryColors: PocketStoriesCategoryColors = PocketStoriesCategoryColors.buildColors(),
```

Comment:
Let's reorder the parameters here. We should ensure `categories` and `categoriesSelections` come before the colors since those are the main parameters for this composable compare to the optional parameter of `categoryColors`.